src/main/java/picard/sam/markduplicates/UmiMetrics.java

```diff
@@ -38,6 +38,8 @@
     private final Histogram<String> observedUmis = new Histogram<>();
     private final Histogram<String> inferredUmis = new Histogram<>();
     private long observedUmiBases = 0;
+    private long observedUmiWithN = 0;
```

Comment:
I think the variable name would make more sense as
observedUmisWithNs

Reply:
changed